files/fr/web/javascript/the_performance_hazards_of_prototype_mutation/index.md

```diff
@@ -7,7 +7,7 @@ tags:
 translation_of: Web/JavaScript/The_performance_hazards_of__[[Prototype]]_mutation
 original_slug: Web/JavaScript/Performance_les_dangers_liés_à_la_modification_de_Prototype
 ---
-{{draft}}
+
```

Comment:
```suggestion
```

Reply:
this was not addressed in my other PR so I will commit this suggested change, thanks!